xml/System.Security.Cryptography/ECDiffieHellmanOpenSsl.xml

```diff
@@ -15,8 +15,16 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides an implementation of the Elliptic Curve Diffie-Hellman (ECDH) algorithm backed by OpenSSL.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+This class should only be used directly when doing platform interop with the system OpenSSL library. When platform interop is not needed, you should use the <xref:System.Security.Cryptography.ECDiffieHellman.Create%2A?displayProperty=nameWithType> factory methods instead of a specific derived implementation.
```

Comment:
Malformed xref. I think it's the `%2A`, and should probably be:

```suggestion
This class should only be used directly when doing platform interop with the system OpenSSL library. When platform interop is not needed, you should use the <xref:System.Security.Cryptography.ECDiffieHellman.Create()?displayProperty=nameWithType> factory methods instead of a specific derived implementation.
```

Reply:
The edit here changed the intent from the method group to a specific overload.  It should be the method group.

Reply:
%2A should work. Reverted.